pkg/utils.py

```diff
@@ -234,6 +237,10 @@ def process_test(
     logger.info("The test %s has started", test["name"])
     fingerprint_index = test["index"]
 
+    global_threshold=0
```

Comment:
This is actually not a global threshold. It should just be within the test's scope as we may have possibilities for multiple tests. I think logic is correct but the naming is confusing.